telegram/contrib/botan.py

```diff
@@ -44,3 +46,23 @@ def track(self, message, event_name='event'):
         except URLError as error:
             self.logger.warn('Botan track error ' + str(error.reason))
             return False
+
+    def shorten(self, url, user_id):
+        if not (url.startswith('http://') or url.startswith('https://')):
```

Comment:
Can you please change to a shorter form:
`if not url.startswith(('http://', 'https://'))`